docs/sdk/api/sdkMiddlewareHttp.md

```diff
@@ -25,12 +25,17 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle HTTP requests for
 3. `includeOriginalRequest` *(Boolean)*: flag whether to include the original request sent in the response. Can be useful if you want to see the final request being sent.
 4. `maskSensitiveHeaderData` *(Boolean)*: flag to mask sensitie data in the header. e.g. Authorization token
 5. `enableRetry` *(Boolean)*: flag to enable retry on network errors and `500` response. (Default: false)
-6. `maxRetries` *(Number)*: number of times to retry the request before failing the request. (Default: 50)
-7. `retryDelay` *(Number)*: amount of milliseconds to wait before retrying the next request. (Default: 200)
+6. `retryConfig` *(Object)*: Field required in the object listed below
+  1. `maxRetries` *(Number)*: number of times to retry the request before failing the request. (Default: 50)
+  2. `retryDelay` *(Number)*: amount of milliseconds to wait before retrying the next request. (Default: 200)
+  3. `backoff` *(Boolean)*: activates exponential backoff. Recommended to prevent spamming of the server. (Default: true)
+  4. `maxDelay` *(Number)*: The maximum duration (milliseconds) to wait before retrying, useful if the delay time grew exponentially more than reasonable
```

Comment:
Useful if used in browser, then you set a maximum time for requests. then you don't have to manually go and calculate what the maximum delay will be based on the `retryDelay` and `maxRetries`